jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -1745,6 +1758,11 @@ function Write-AdmuErrorMessage {
 
             $Script:ErrorMessage = "User Creation Error. Click the link below for troubleshooting information."
         }
+        "user_folder_error" {
+            Write-ToLog -Message:("User Folder Error: One of the user's main folder (Desktop, Downloads, Documents, Favorites, Pictures, Videos, Music) path is redirected. Verify that the user's main folders path are default and not redirected to another path (ie. Network Drive). Please refer to this link for more information: https://github.com/TheJumpCloud/jumpcloud-ADMU/wiki/troubleshooting-errors") -Level Error
```

Comment:
Thank you for this, I'm glad to see us making use of that work we did to add better logging and error messages.